src/OrchardCore.Modules/OrchardCore.Media.AmazonS3/AwsStorageOptionsConfiguration.cs

```diff
@@ -50,7 +50,8 @@ private void ParseBucketName(AwsStorageOptions options, TemplateContext template
 
             options.BucketName = template
                 .Render(templateContext, NullEncoder.Default)
-                .Replace("\r", String.Empty).Replace("\n", String.Empty);
+                .Replace(System.Environment.NewLine, String.Empty)
```

Comment:
Last thing add `using System` instead

Reply:
There is a conflict in Environment naming between OrchardCore.Environment and System, so it's either : 
System.Environment.NewLine
or 
using static System.Environment;
and
```C#
 options.BucketName = template
                .Render(templateContext, NullEncoder.Default)
                .Replace("\r", String.Empty).Replace("\n", String.Empty);
                .Replace(NewLine, String.Empty)
```